src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+function numberWithCommas(x) {
+  return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+}
+
+const popu = document.querySelectorAll('.population');
```

Comment:
try to use clear variable names

```suggestion
const population = document.querySelectorAll('.population');
```